octopod-backend/src/Octopod/Server/Logging.hs

```diff
@@ -0,0 +1,74 @@
+-- |
+--Module      : Octopod.Server.Logging
+--Description : Octopod Server logging setup
+module Octopod.Server.Logging
+  ( runLog,
+    LogConfig (..),
+    show',
+    logLocM,
+    logStr,
+    katipAddContext,
+    katipAddNamespace,
+    Severity (..),
+    KatipContext,
+    KatipContextT,
+    LogItem (..),
+    ToObject (..),
+    PayloadSelection (..),
+  )
+where
+
+import Control.Exception
+import Control.Lens hiding (scribe)
+import Control.Monad
+import Control.Monad.Trans.Control
+import Data.List (intersperse)
+import Data.Text.Lazy.Builder
+import GHC.Generics
+import Katip
+import Katip.Core
+import Katip.Scribes.Handle
+import System.IO
+import Types hiding (stdout)
+
+data LogConfig = LogConfig
+  { project :: ProjectName
+  , debug :: Bool
+  , minimal :: Bool
+  }
+  deriving stock (Generic)
+
+makeLogEnv :: LogConfig -> IO LogEnv
+makeLogEnv conf = do
+  scribe <- mkHandleScribeWithFormatter formatter ColorIfTerminal stdout (permitItem severity) V3
+  env <- initLogEnv "octopod" $ Environment $ uProjectName $ conf ^. #project
+  env' <- registerScribe "stdout" scribe defaultScribeSettings env
+  pure env'
+  where
+    severity = if conf ^. #debug then DebugS else InfoS
+    formatter :: LogItem a => ItemFormatter a
+    formatter =
+      if not $ conf ^. #minimal
+        then bracketFormat
+        else \_color _verb item ->
```

Comment:
I'm not sure I like the current "minimal" format. I would prefer for it to maintain the bracket syntax that `bracketFormat` has, but with only the essential fields.

Furthermore, I think printing the current time is useful (maybe not the date though)
Namespace seems useless when developing locally -- just noise.